src/Ombi.Notifications/Agents/DiscordNotification.cs

```diff
@@ -21,12 +22,34 @@ public class DiscordNotification : BaseNotification<DiscordNotificationSettings>
         public DiscordNotification(IDiscordApi api, ISettingsService<DiscordNotificationSettings> sn,
                                    ILogger<DiscordNotification> log, INotificationTemplatesRepository r,
                                    IMovieRequestRepository m, ITvRequestRepository t, ISettingsService<CustomizationSettings> s)
-            : base(sn, r, m, t,s,log)
+            : base(sn, r, m, t, s, log)
         {
             Api = api;
             Logger = log;
+            ShowCompactEmbed = new Dictionary<NotificationType, bool>();
+            // Temporary defaults
+            ShowCompactEmbed.Add(NotificationType.NewRequest, true);
```

Comment:
All of these should be driven from the UI

Reply:
Implemented all the other changes except this as I'm not sure how to do implement that.